cookie-banner-rules-list.json

```diff
@@ -540,34 +537,31 @@
     },
     {
       "click": {
-        "optIn": "button.btn-accept",
-        "optOut": "button.btn-reject",
-        "presence": "div#cookie-disclosure"
+        "optIn": "#cookie-disclosure-accept",
+        "optOut": "#cookie-disclosure-reject",
+        "presence": "#cookie-disclosure"
       },
       "cookies": {},
       "id": "6037802d-9a37-4df2-bf35-9ad60c478725",
       "domains": ["netflix.com"]
     },
     {
       "click": {
-        "optIn": "div[role=dialog] > div > button",
-        "optOut": "div[role=dialog] > div > button + button",
-        "presence": "#scrollview + div"
+        "optIn": "._a9--._a9_0",
+        "optOut": "._a9--._a9_1",
+        "presence": ".x1n2onr6.xzkaem6"
```

Comment:
This is slightly better than the old rule (which could have targeted any dialog with that structure). But I'm still worried that this isn't specific enough to the cookie banner. Also it may break when their front-end is redeployed.
I suggest that we try to find a cookie rule instead. If that doesn't work let's remove the rule entirely.